source/index.md

```diff
@@ -52,162 +61,1289 @@ The Horus API does a division of authentication according to the type of user, W
-
- Route: `GET "/api/v1/manager/clients"`
-
- Command Line: `horus-cli list clients`
-
-* Show
-
- You also can show a resource using a command line, with the id of the resource, or accessing a route, both cases are described below. In this exeample we use the manager.
-
- Route: `GET "/api/v1/manager/clients/:id"`
-
- Command Line: `horus-cli show clients "id"`
+Once authenticated, except a user, you can create new resources, edit, list and show it through the following command lines.
+
+### Create
+
+You can create a resource using a command line or accessing a route, both cases are described below. In this example we use the manager.
+
+Route: `POST "/api/v1/manager/clients"`
+
+> Create Resource
+
+```ruby
+horus-cli create clients '"corporate-name":"Corporate_name", "trade-name":"Trade_name", "key-name":"key", "email":"client@example.com"'
+
+# Return
+
+{
+  "corporate-name":"Corporate_name", 
+  "trade-name":"Trade_name", 
+  "key-name":"key", 
+  "email":"client@example.com", 
+  "type":"clients", 
+  "id":"497cec48-f018-4382-a5ea-f9e0afbec09a"
+}
+```
+
+```shell
+# The access_token is obtained in the login.
+
+curl --header "Content-Type: application/vnd.api+json" -H "Authorization: Bearer 'access_token'" -X POST -d  
+'{
+  "data":
+  {
+    "type":"clients", 
+    "attributes":
+    {
+      "corporate-name":"Corporate_name", 
+      "trade-name":"Trade_name", 
+      "key-name":"key", 
+      "email":"client@example.com"
+    }
+  }
+}'  
+http://localhost:3000/api/v1/manager/clients
+
+# Return
+
+{
+  "data":
+  {
+    "id":"fec0f2a6-ac7c-4e51-94e3-9ba222b0e467", 
+    "type":"clients", 
+    "links":
+    {
+      "self":"http://localhost:3000/api/v1/manager/clients/fec0f2a6-ac7c-4e51-94e3-9ba222b0e467"
+    }, 
+    "attributes":
+    {
+      "corporate-name":"Corporate_name", 
+      "trade-name":"Trade_name", 
+      "key-name":"key", 
+      "email":"client@example.com"
+    }, 
+    "relationships":
+    {
+      "client-address":
+      {
+        "links":
+        {
+          "self":"http://localhost:3000/api/v1/manager/clients/fec0f2a6-ac7c-4e51-94e3-9ba222b0e467/relationships/client-address", 
+          "related":"http://localhost:3000/api/v1/manager/clients/fec0f2a6-ac7c-4e51-94e3-9ba222b0e467/client-address"
+        }, 
+        "data":null
+      }, 
+      "client-telephones":
+      {
+        "links":
+        {
+          "self":"http://localhost:3000/api/v1/manager/clients/fec0f2a6-ac7c-4e51-94e3-9ba222b0e467/relationships/client-telephones", 
+          "related":"http://localhost:3000/api/v1/manager/clients/fec0f2a6-ac7c-4e51-94e3-9ba222b0e467/client-telephones"
+        }
+      }
+    }
+  }
+}
+```
+
+### Update
+
+You also can update a resource using a command line, with the id of the resource, or accessing a route, both cases are described below. In this exeample we use the manager.
+
```

Comment:
Correct: "example" not 'exeample' : You also can update a resource using a command line, with the id of the resource, or accessing a route, both cases are described below. In this example we use the manager.